src/hooks/useWithdrawFormState.ts

```diff
@@ -37,17 +37,16 @@ export type WithdrawFormAction = {
   fieldName: FormFields | "reset"
   address?: string
   value: string
+  max?: BigNumber //TODO: get underlying token balance from registry pool
```

Comment:
Can you use registry pool and get the underlying token balance?

Reply:
On `BasicPoolsProvider.tsx`, we are getting underlyingTokenBalances on line 249.

Reply:
great, is there a problem with using pools from registry then?

Reply:
Should the TODO: comment still be there?

Reply:
Yes, on line 112, we are getting pools from getPoolsBaseData on BasicPoolsProvider.
So, underlyingTokenBalance is Zero now.